components/access-management/OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
+  - kunmingg
```

Comment:
Remove @kunmingg; @kunmingg is focused on other things so this is all you for the time being.

Also don't duplicate approvers under reviewers; an approver is automatically a reviewer.